src/easyreflectometry/calculators/calculator_base.py

```diff
@@ -177,3 +177,16 @@ def sld_profile(self, model_id: str) -> tuple[np.ndarray, np.ndarray]:
 
     def set_resolution_function(self, resolution_function: Callable[[np.array], np.array]) -> None:
         return self._wrapper.set_resolution_function(resolution_function)
+
+    @property
+    def magnetism(self):
```

Comment:
`magnetism` is not immediately obvious to be a boolean. I would think `self.magnetism` contains some object/dict/structure with the description of magnetic properties.
Maybe `has_magnetism`? or `is_magnetic` if it makes physical sense.

Reply:
Changed to `include_magnetism`